<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WEventoPaginaFinal</class>
 <widget class="QWizardPage" name="WEventoPaginaFinal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Revisa los datos antes de crear el evento</string>
  </property>
  <property name="subTitle">
   <string>Comprueba que los datos son correctos; de no ser así, vuelve atrás y modificalos a tu gusto.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>¿Son los siguientes datos correctos?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>El evento</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="4">
        <widget class="QLineEdit" name="eventoLine">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	background-color: rgb(250, 255, 150);
}</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Se celebrará los días</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="diasLine">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	background-color: rgb(250, 255, 150);
}</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>de</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="mesesLine">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	background-color: rgb(250, 255, 150);
}</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>del año</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="4">
        <widget class="QLineEdit" name="yearLine">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	background-color: rgb(250, 255, 150);
}</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Puede añadir algún comentario final (opcional)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="comentBox"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
